zcash_keys/src/keys.rs

```diff
@@ -601,9 +633,43 @@ impl UnifiedAddressRequest {
     }
 }
 
+/// Errors that may be returned from the [UnifiedFullViewingKey] and [UnifiedIncomingViewingKey] types.
+#[derive(Debug)]
+pub enum UnifiedError {
```

Comment:
Naming bikeshed: `UnifiedError` doesn't mean very much to me, maybe `DerivationError` would be better? Or `KeyDerivationError`?

Reply:
I'm glad you brought it up. I wasn't settled on it either. There is another enum error type in this file--is it worth combining them?

Reply:
Oh, right, yes, the existing `DerivationError` type should just be modified.